packages/fuel-gauge/src/transaction-response.test.ts

```diff
@@ -1,4 +1,4 @@
-import type {} from '@fuel-ts/account/dist/providers/__generated__/operations';
+import type { } from '@fuel-ts/account/dist/providers/__generated__/operations';
```

Comment:
1. IMO we shouldn't allow empty imports,
2. This import is from the `/dist/` folder, which shouldn't happen (just an observation, it was there previously, you don't need to tackle it here)